src/d3a/d3a_core/live_events.py

```diff
@@ -46,6 +49,20 @@ def __init__(self, area_uuid, area_params):
     def apply(self, area):
         if area.uuid != self.area_uuid:
             return False
+        if key_in_dict_and_not_none(self.area_params, 'type'):
+            if self.area_params['type'] == "MarketMaker":
+                strategy_obj = MarketMakerStrategy
+                del self.area_params['type']
+                del self.area_params['energy_rate_profile_uuid']
+                self.area_params = {'strategy_object': strategy_obj(**self.area_params)}
+            elif self.area_params['type'] == "InfiniteBus":
+                strategy_obj = InfiniteBusStrategy
+                del self.area_params['energy_rate_profile_uuid']
+                del self.area_params['buying_rate_profile_uuid']
+                self.area_params = {'strategy_object': strategy_obj(**self.area_params)}
+
+            del self.area_params['type']
+            del self.area_params['number_of_clones']
```

Comment:
Can you explain why this is needed? This reminds me actually, to remove these already in the d3a-web.
But even now: if there is no "number_of_clones" member, this will fail, so i suggest to remove this line, if you can live without it (and maybe create a small PR that deals with this in the d3a-web) Thanks

Reply:
Agree with @hannesdiedrich , and in order to circumvent the KeyError here, you can use the pop method: 
`self.area_params.pop('number_of_clones', None)`

Reply:
Could you please add another story to deprecate `number_of_clones` in d3a_web (at least the sending of this parameter to the d3a, which was never needed). Thanks!

Reply:
https://gridsingularity.atlassian.net/browse/D3ASIM-2989
Done!